spec/example_app/app/dashboards/order_dashboard.rb

```diff
@@ -14,6 +14,8 @@ class OrderDashboard < Administrate::BaseDashboard
     line_items: Field::HasMany,
     total_price: Field::Number.with_options(prefix: "$", decimals: 2),
     shipped_at: Field::DateTime,
+    status: Field::Enum.with_options(
+      enum: { "pending" => 0, "confirmed" => 1 }),
```

Comment:
I am also not thrilled about it but the Field class is all we have available at the time of search construction. The instantiated object of that class isn't available until we're actually showing something in the view. Another option would be to annotate the Field class with which AR class it is for, as this would give us the ability to reflect on the enum keys/values from the Field class context. I think that might be a pretty big api change though.